packages/cli/src/zosjobs/list/jobs/Jobs.definition.ts

```diff
@@ -58,6 +64,19 @@ export const JobsDefinition: ICommandDefinition = {
             {
                 options: "--rff jobid --rft table",
                 description: "List all jobs with default owner and prefix settings, displaying only the job ID of each job"
+            },
+            {
+                options: "--ed=true",
```

Comment:
The `=true` is unnecessary. For this first example, it might be good to spell out the command option name instead of using the alias.